index.js

```diff
@@ -37,12 +37,16 @@ module.exports = {
       return failBuild(`** Static HTML export next.js projects do not require this plugin **`)
     }
 
-    // TO-DO: check scripts to make sure the app isn't manually running NoN
-    // For now, we'll make it clear in the README
-    // const isAlreadyUsingNextOnNetlify = Object.keys(dependencies).find((dep) => dep === 'next-on-netlify');
-    // if (isAlreadyUsingNextOnNetlify) {
-    //   return failBuild(`This plugin cannot support apps that manually use next-on-netlify. Uninstall next-on-netlify as a dependency to resolve.`);
-    // }
+    const hasNextOnNetlifyInstalled = Object.keys(dependencies).find((dep) => dep === 'next-on-netlify')
+    const hasNextOnNetlifyPostbuildScript = Object.keys(scripts).find(
+      (script) => script === 'postbuild' && scripts[script].includes('next-on-netlify'),
+    )
```

Comment:
Would the following be simpler?

```js
    const hasNextOnNetlifyPostbuildScript = typeof scripts.postbuild === 'string' && scripts.postbuild.includes('next-on-netlify')
```

Reply:
yes and more reliable, idk why i clung to Object.keys and somehow forgot object values can be accessed at object[key] lul ty